src/room/GroupCallView.tsx

```diff
@@ -495,9 +495,7 @@ export const GroupCallView: FC<Props> = ({
     }
   } else if (left && widget !== null) {
     // Left in widget mode:
-    if (!returnToLobby) {
-      body = null;
-    }
+    body = returnToLobby ? lobbyView : null;
   } else if (preload || skipLobby) {
```

Comment:
How has the previous version worked if we need to set it to `lobbyView` in the widget case?

Reply:
It worked because the `else` branch at the bottom of this condition tree used to be [outside of the condition tree](https://github.com/element-hq/element-call/blob/9b59544db29c09d5953649c9b15a0b40744b62dd/src/room/GroupCallView.tsx#L424), so returnToLobby=true would cause this to fall through to the `else` case and return the lobby

Reply:
oh I see so this was not working after the change and until now. Cool Thanks for the explanation!